server/api/controllers/article.js

```diff
@@ -188,3 +196,77 @@ export const dislikeArticle = async (req, res) => {
     });
   }
 };
+
+/**
+  * @export
+  * @function createHighlight
+  * @param {Object} req - request received
+  * @param {Object} res - response object
+  * @returns {Object} JSON object (JSend format)
+  */
+export const createHighlight = async (req, res) => {
+  try {
+    const { params: { id: articleId }, user: { id: readerId }, body: { highlight, comment } } = req;
+    const newHighlight = await HighlightComment.create({
+      articleId,
+      readerId,
+      highlight,
+      comment
+    });
+
+    return res.status(201).send({
+      status: 'success',
+      data: {
+        message: 'You highlighted successfully!',
+        highlight: newHighlight
+      }
+    });
+  } catch (e) {
+    return res.status(500).send({
+      status: 'error',
+      message: 'Internal server error occured.'
+    });
+  }
+};
+
+/**
+  * @export
+  * @function getHighlights
+  * @param {Object} req - request received
+  * @param {Object} res - response object
+  * @returns {Object} JSON object (JSend format)
+  */
+export const getHighlights = async (req, res) => {
+  try {
+    const { params: { id: articleId }, user: { id: readerId } } = req;
+    const foundHighlights = await HighlightComment.findAll({
+      attributes: ['articleId', 'readerId', 'highlight', 'comment'],
+      where: {
+        articleId: { [Op.eq]: articleId },
+        readerId: { [Op.eq]: readerId }
+      }
+    });
+
+    if (foundHighlights.length === 0) {
```

Comment:
@Ibidapo Please change line `250` to `if (!foundHighlights.length) {...}`.

Reply:
Noted!